src/pages/PhotoInAlbumPage.tsx

```diff
@@ -44,8 +44,10 @@ function PhotoInAlbumPage({ toggleHeader }: PhotoInAlbumProps) {
   if (isError) {
     return <div>{error ? error.message : "Unknown error"}</div>
   }
-
-  return <Photo photo={data} albumId={albumId} />
+  if (data) {
+    return <Photo photo={data} albumId={albumId} />
+  }
+  return <h1>No Photos here</h1>
```

Comment:
Let's remove `PhotoInAlbumPage` altogether, can combine with *Photo.tsx*.

Reply:
I thought exactly the  same, will do! :)